atlasdb-cassandra/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/pool/CassandraClientPoolMetricsTest.java

```diff
@@ -57,6 +58,18 @@ public void metricsAreProducedAndFiltered() {
                 .containsKey(createMeanActiveTimeMillisMetric("mean"));
     }
 
+    @Test
+    public void recordPoolSizeCorrectlySetsSize() {
```

Comment:
nit: maybe `recordPoolSizeSetsSizeToLastReportedValue` to explain what correct means?